app/Jobs/SendPostToCustomerIo.php

```diff
@@ -21,14 +21,22 @@ class SendPostToCustomerIo implements ShouldQueue
      */
     protected $post;
 
+    /**
+     * Whether or not to log that this Post was sent to Quasar.
```

Comment:
So below it looks like we are logging that we sent to blink, and then here we say that we are logging if we are sending to quasar, and the Job implies we are sending to CustomerIO. It is sorta confusing, because I know things go through blink and then to other platforms, but I think we should be consistent. I guess this should just be "sending to customer.io" since that is where it ends up?

Reply:
ah yes! Thanks for catching that! Updated :)